gotham/src/.rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 102
```

Comment:
Without this, rustfmt emits errors because it doesn't know how to fold one of the trait bounds, but without folding, it can't be made less that 100 characters. Manual folding gets put back like it was...

Reply:
c.f. https://github.com/rust-lang-nursery/rustfmt/issues/1080